auction-app-backend/src/main/java/com/example/auctionapp/util/LevenshteinDistance.java

```diff
@@ -0,0 +1,43 @@
+package com.example.auctionapp.util;
+
+public class LevenshteinDistance {
+    public static int computeLevenshteinDistance(CharSequence lhs, CharSequence rhs) {
+        int len0 = lhs.length() + 1;
+        int len1 = rhs.length() + 1;
+
+        // the array of distances
+        int[] cost = new int[len0];
+        int[] newcost = new int[len0];
+
+        // initial cost of skipping prefix in String s0
+        for (int i = 0; i < len0; i++) cost[i] = i;
+
+        // dynamically computing the array of distances
```

Comment:
Remove empty space 
No need for this 
// comment 
empty line 
// another comment